packages/chatbot-server-mongodb-public/src/processors/generateResponseWithTools.test.ts

```diff
@@ -551,10 +523,13 @@ describe("generateResponseWithTools", () => {
       });
 
       test("should handle streaming with guardrail rejection", async () => {
-        const mockDataStreamer = makeMockDataStreamer();
         const generateResponse = makeGenerateResponseWithTools({
           ...makeGenerateResponseWithToolsArgs(),
-          inputGuardrail: makeMockGuardrail(false),
+          inputGuardrail: async () => {
+            // sleep for 2 seconds
```

Comment:
```suggestion
            // sleep for 5 seconds
```

also previous comment abt timeout length